src/abe_policy/policy.rs

```diff
@@ -108,7 +88,11 @@ impl Policy {
     pub fn remove_attribute(&mut self, attr: &Attribute) -> Result<(), Error> {
         if let Some(dim) = self.dimensions.get_mut(&attr.dimension) {
             if dim.nb_attributes() == 1 {
-                self.remove_dimension(&attr.dimension)
+                // self.remove_dimension(&attr.dimension)
```

Comment:
dead code

Reply:
I left it because it will need to be un-commented when the neutral attribute will be released
Confirm if you want me to remove it anyway

Reply:
No problem to keep it if you clearly document (via a code comment) that it needs to be added back. You can add a reference to the issue raised from a comment above.